zhaquirks/inovelli/__init__.py

```diff
@@ -66,97 +69,42 @@
 # 1 - down button
 # 2 - up button
 # 3 - config button
+# 4 - aux down button
+# 5 - aux up button
+# 6 - aux config button
 
-BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3}
+BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3, 4: BUTTON_4, 5: BUTTON_5, 6: BUTTON_6}
 ON = "Up"
 OFF = "Down"
 CONFIG = "Config"
+AUX_ON = "Aux Up"
+AUX_OFF = "Aux Down"
+AUX_CONFIG = "Aux Config"
 
 NOTIFICATION_TYPE = "notification_type"
 
 
-class Inovelli_VZM31SN_Cluster(CustomCluster):
-    """Inovelli VZM31-SN custom cluster."""
-
-    cluster_id = 0xFC31
-    name = "InovelliVZM31SNCluster"
-    ep_attribute = "inovelli_vzm31sn_cluster"
+class Inovelli_Cluster(CustomCluster):
+    """Inovelli base cluster."""
 
     attributes = {
         0x0001: ("dimming_speed_up_remote", t.uint8_t, True),
```

Comment:
Not for this PR, but we need to switch to the new zigpy `AttributeDefs` style at some point.

Reply:
Again, no need to do it in this PR already, but saving a code sample here for reference how it would look:
https://github.com/zigpy/zha-device-handlers/blob/ab0d248defda933019c98ddab451b5969fe99c16/zhaquirks/legrand/cable_outlet.py#L42-L59